strongbox-data-service/src/main/java/org/carlspring/strongbox/config/HazelcastConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package org.carlspring.strongbox.config;
+
+import org.carlspring.strongbox.data.CacheName;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.EvictionPolicy;
+import com.hazelcast.config.MapConfig;
+import com.hazelcast.config.MaxSizeConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Configuration
+public class HazelcastConfiguration
+{
+
+    @Bean
+    public HazelcastInstance hazelcastInstance(Config config)
+    {
+        return Hazelcast.newHazelcastInstance(config);
+    }
+
+    @Bean
+    public Config hazelcastConfig()
+    {
+        final Config config = new Config()
+                                      .setInstanceName("hazelcast-instance")
```

Comment:
This should be `strongbox`. This leads me to the thought that Strongbox should have instance names (I've raised #693 for this).